discord_thread_summarizer_using_openai/discord_utils.bal

```diff
@@ -0,0 +1,55 @@
+import ballerina/time;
+import ballerina/http;
+
+const time:Seconds DISCORD_EPOCH = 1420070400000.00;
+
+http:Client discordClient = check new (DISCORD_API_URL);
+
+function getChannelDetails(int channelId) returns Channel|error {
+    string path = string `/channels/${channelId}`;
+    http:Response response = check discordClient->get(path, headers = {
```

Comment:
Rather than doing this, shall we check whether we cn use built-in auth support in ballerina HTTP client ?
- https://ballerina.io/learn/by-example/http-client-bearer-token-authentication/

Reply:
I am getting an unauthorized error when trying the following code block.
```
final http:Client discordClient = check new (DISCORD_API_URL,
    auth = {
        token: botToken
    }
);
```


Note that the expected header should be
`"Authorization": "Bot <token>"` not the standard `"Authorization": "Bearer <token>"`

How can we set this in the client. @ayeshLK

Reply:
It seems they are using a different approach rather than the standard bearer-token. If there is no alternative for this, then use the manual header setting.